### DISCUSSION TITLE
Support platformAutoMerge on bitbucket-server

### DISCUSSION THREAD

**Discussion by surskitt** [renovatebot/renovate#32638] [Suggest an Idea]
At the moment platformAutoMerge is only supported by azure, gitea, github and gitlab. Since version 8.15, bitbucket server has supported automatically merging pull requests once they have been approved.

https://confluence.atlassian.com/bitbucketserver/merge-a-pull-request-808488562.html#Mergeapullrequest-Auto-mergeapullrequest

https://confluence.atlassian.com/bitbucketserver/merge-a-pull-request-808488562.html#Mergeapullrequest-Auto-mergeapullrequest

Would it be possible to use this feature to allow using platformAutoMerge with bitbucket-server?

**Comment by rarkins** [comment]
You have referenced a UI feature, need to see if it's also in the API.

**Reply by RahulGautamSingh** [reply]
API details here: https://developer.atlassian.com/server/bitbucket/rest/v1000/api-group-pull-requests/#api-api-latest-projects-projectkey-repos-repositoryslug-pull-requests-pullrequestid-merge-post

**Reply by RahulGautamSingh** [reply]
Issue Created: https://github.com/renovatebot/renovate/issues/40451
